### ISSUE TITLE
Bolt\error\ConnectException: Requested Protocol version (80.84.84.72) not yet implemented

### ISSUE THREAD

**Issue by ENGAHMAD97** [neo4j-php/neo4j-php-client#132]
when I try to run a query I get this error

Bolt\error\ConnectException: Requested Protocol version (80.84.84.72) not yet implemented

$client = ClientBuilder::create()
            ->withDriver('bolt', 'bolt+s://xxxx:7687/')
            ->withDriver('neo4j', 'neo4j://xxxxx:7474/', Authenticate::basic('xxx', 'xxxxx'))
            ->withDriver('https', 'xxxxx:7474/')
            ->withDefaultDriver('neo4j')
            ->build();

any help?

**Comment by transistive** [comment]
Yes, the neo4j protocol typically runs on port 7687. 7474 is for http. If that doesnt work I can double check later tonight!

Good luck

Sent from ProtonMail mobile

-------- Original Message --------
